CONTRIBUTING.md

```diff
@@ -61,7 +61,14 @@ Pull requests
 
 When creating pull request, please heed the following:
 
-* Base your code on the latest master branch to avoid manual merges
-* Code review may ensue in order to help shape your proposal
+* Base your code on the latest `master` branch to avoid manual merges
+  If necessary, stable versions (branches `stable/<major>.<minor>`) may be updated
+  by a team member with a cherry-pick after a pull-request is merged.
+  This means a pull-request target is always the `master` branch.
+* Code review may ensure in order to help shape your proposal
 * Pull request must adhere to 2-Clause BSD licensing
 * Explain the problem and your proposed solution
+* If applicable cite the issue(s) number(s) in your pull-request description,
+    for example `Fixes: #1234`, `Closes: #1234`, or `Ref: #1234`.
+* Read [README.md](./README.md) to learn about the commands to shape your code
+  You may want to run `make style` before submitting your work.
```

Comment:
```suggestion
```

I'd leave this out to avoid cross-referencing in a loop. Pointing to readme above where the style part is explained suffices. We are not super strict and phpcb does most of the work post-merge anyway.